commit 74ad7f98c5804beae6fb4a271adc02d85bb62cee
Author: Rick Poyner <rick.poyner@tri.global>
Date:   Tue Mar 3 13:31:30 2026 -0500

    wip

diff --git include/nanobind/ndarray.h include/nanobind/ndarray.h
index b28d588..8142c2a 100644
--- include/nanobind/ndarray.h
+++ include/nanobind/ndarray.h
@@ -22,7 +22,9 @@ NAMESPACE_BEGIN(NB_NAMESPACE)
 NAMESPACE_BEGIN(dlpack)
 
 enum class dtype_code : uint8_t {
-    Int = 0, UInt = 1, Float = 2, Bfloat = 4, Complex = 5, Bool = 6,
+    Int = 0, UInt = 1, Float = 2,
+    OpaqueHandle = 3,
+    Bfloat = 4, Complex = 5, Bool = 6,
     Float8_E3M4 = 7, Float8_E4M3 = 8, Float8_E4M3B11FNUZ = 9,
     Float8_E4M3FN = 10, Float8_E4M3FNUZ = 11, Float8_E5M2 = 12,
     Float8_E5M2FNUZ = 13, Float8_E8M0FNU = 14,
